### PR TITLE
Partial #1559 #5

### PR DESCRIPTION
**Pull request by grimreaper**

### CODE REVIEW

**Comment by thpr** [comment]
Can we get these to where they can all be viewed in github?  1100 still too many

File: code/src/java/pcgen/core/PlayerCharacterUtilities.java [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+	 */
+	static void setProf(final Equipment equip, final Equipment eqm)
+	{
+		eqm.put(ObjectKey.WEAPON_PROF, equip.get(ObjectKey.WEAPON_PROF));
+		// In case this is used somewhere it shouldn't be used,
+		// set weight and cost to 0
+		eqm.put(ObjectKey.WEIGHT, BigDecimal.ZERO);
+		eqm.put(ObjectKey.CURRENT_COST, BigDecimal.ZERO);
+	}
+}
```
Comment (cpmeister) [inline code/src/java/pcgen/core/PlayerCharacterUtilities.java]: This doesn't belong in this PR

**Comment by grimreaper** [comment]
bad merge fixed
